packages/components/file-upload/multiple-file-upload.component.html

```diff
@@ -2,6 +2,7 @@
     kbqFileDrop
     class="kbq-file-upload"
     [class.kbq-disabled]="disabled"
+    [class.kbq-error]="errorState"
```

Comment:
мб здесь аналогично нужно добавить?
```suggestion
    [class.kbq-error]="invalid"
```
и
```
get invalid(): boolean {
    return files.some(({invalid}) => invalid) || this.errorState;
}
```

Reply:
тут немного хитрее, потому что в multiple если отдельный файл по валидации не прошел, то только он подсветится.

но с идеей того чтобы в multiple и single я согласен